CollAction/ValidationAttributes/DateWithinMonthsAttribute.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Microsoft.AspNetCore.Http;
+using System;
+using System.ComponentModel.DataAnnotations;
+using System.Reflection;
+
+namespace CollAction.ValidationAttributes
+{
+    public class DateWithinMonthsAttribute : ValidationAttribute
+    {
+        private readonly int _months;
+
+        public String OfDateProperty { get; set; }
+
+        public DateWithinMonthsAttribute(int months)
```

Comment:
Ah, I see your point about leap years. Sure, go ahead and cell in.

There are workarounds for the primitive type issue btw: https://msdn.microsoft.com/en-us/library/system.configuration.timespanvalidatorattribute(v=vs.110).aspx .